gpustack/api/exceptions.py

```diff
@@ -13,6 +13,23 @@ def __init__(self, status_code: int, reason: str, message: str):
         self.message = message
 
 
+def http_exception_handler(request: Request, exc: HTTPException):
+    return JSONResponse(
+        status_code=exc.status_code,
+        content={
+            "code": exc.status_code,
+            "reason": exc.reason,
+            "message": exc.message,
+        },
+    )
+
+
+app = FastAPI()
+
+
+app.add_exception_handler(HTTPException, http_exception_handler)
```

Comment:
We have an exception handler registered in https://github.com/gpustack/gpustack/blob/main/gpustack/server/app.py#L31